recipes/serpent/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.12" %}
+
+package:
+  name: serpent
+  version: {{ version }}
+
+source:
+  fn: serpent-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/serpent/serpent-{{ version }}.tar.gz
+  md5: 05869ac7b062828b34f8f927f0457b65
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - serpent
+
+about:
+  home: https://github.com/irmen/Serpent
+  license: MIT
+  summary: 'Serialization based on ast.literal_eval'
+
+extra:
+  recipe-maintainers:
+    - whart222
+    - juanlu001
```

Comment:
I would not worry much, https://github.com/juanlu001/ shows my profile. It's trivial anyway :)